.github/tools/gitchanges.py

```diff
@@ -49,9 +49,14 @@ def getTags():
     tags = filter(lambda x: "-" not in x, tags)
     return natsorted(tags,reverse=True)
 
-def getCommitsBetween( tagA, tagB = "HEAD" ):
-    commits = os.popen(f"git -P shortlog {tagA}..{tagB}").read().strip().splitlines()
-    return "\n".join(map( lambda x: x.replace("      ", " - "), commits ))
+def getCommitsBetween( tagA, tagB = "HEAD", dir=None):
+    if dir:
+        original_dir = os.getcwd()
+        os.chdir(dir)
+    commits = os.popen(f'git log --format="%s (by %an)" --no-merges --reverse {tagA}..{tagB}').read().strip().splitlines()
```

Comment:
There was no way to change `shortlog` to not group commits, so I've replaced it with `git log --no-merges`. Reverse was needed because that's the order used by `shortlog`, which I guess makes sense with older commits first, as we read top to bottom.

The format includes the git author name at the end of the commit title, in case we wanted to still provide attribution.
`%s` -> commit title
`%an` -> commit author name